build/build.cmd

```diff
@@ -0,0 +1,12 @@
+IF "%buildConfiguration%"=="" (SET configuration=Debug) ELSE (SET configuration=%buildConfiguration%)
+
+nuget restore Datadog.Trace.sln
+nuget restore src\OpenTelemetry.DynamicActivityBinding\OpenTelemetry.DynamicActivityBinding.sln
+msbuild Datadog.Trace.proj /t:BuildCsharp /p:Configuration=%configuration%
+dotnet pack -c %configuration% -o src\bin\SignalFx.Tracing src\Datadog.Trace\Datadog.Trace.csproj
+dotnet pack -c %configuration% -o src\bin\SignalFx.Tracing.OpenTracing src\Datadog.Trace.OpenTracing\Datadog.Trace.OpenTracing.csproj
```

Comment:
Should the SignalFx.Tracing references be in here? This might have been an artifact of some local development/testing.